### DISCUSSION TITLE
Does polishing still have a polishing cap?

### DISCUSSION THREAD

**Discussion by rafaderren** [HelixNebulaStudio/RiseOfTheDead#617] [Q&A] [answered]
Whenever I try to get a weapon below 0.01 it just fails. I have spent 1.5k perks for it and still no single change.
Zakku tried to find out if it has a cap on dev branch and seems like the polishing still fails.
![image](https://github.com/user-attachments/assets/631035e6-eb32-4895-919e-d9733296310c)

**Comment by MXKhronos** [comment, marked as answer]
no, there isn't a cap, it's just the lower your float, the harder it is to reduce even more when you polish, you get a reduction of 0.01 to 0.2, that random number subtracts your float and if it's below 0, it fails, there's also a chance that a seed doesn't exist to match the potential float hence it can fail that way too after 64 tries.
